core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/operators/aggregate/PartialAggregateOpExec.scala

```diff
@@ -53,7 +53,7 @@ class PartialAggregateOpExec[Partial <: AnyRef](
       case Right(_) =>
         partialObjectPerKey.iterator.map(pair => {
           val fields: Array[Object] = (pair._1 :+ pair._2).toArray
-          Tuple.newBuilder().add(schema, fields).build()
+          Tuple.newBuilder(schema).addSequentially(fields).build()
```

Comment:
same comments here

Reply:
similar comment as finalAggOpExec, this should be good for now

Reply:
Same as https://github.com/Texera/texera/pull/1166#discussion_r659220480